src/Swagger.ObjectModel/Builders/PathItemBuilder.cs

```diff
@@ -11,15 +13,15 @@ namespace Swagger.ObjectModel.Builders
     /// </summary>
     public class PathItemBuilder
     {
-        /// <summary>
-        /// The operations.
-        /// </summary>
-        private IDictionary<HttpMethod, Operation> operations;
+        private readonly List<Parameter> parameters = new List<Parameter>();
+        private readonly Operation operation = new Operation();
+        private readonly string method;
+
+        public PathItemBuilder(string method)
+        {
+            this.method = method;
```

Comment:
Use the HttpMethod enumeration here... It will be a lot easier to use than magic strings for the method